src/site/content/en/how-to-optimize-inp/find-slow-interactions-in-the-field/index.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
**Note:** This post has already underwent extensive technical review in Google Docs, and should not be further reviewed for anything other than incompatibility with our style guide.